recipe/meta.yaml

```
@@ -46,12 +47,12 @@ test:
 about:
   home: https://web.mit.edu/kerberos/
   license: MIT
-  license_file: doc/notice.rst
   license_family: MIT
-  summary: 'A network authentication protocol.'
+  license_file: doc/notice.rst
+  summary: A network authentication protocol.
   description: |
     Kerberos is a network authentication protocol. It is designed to provide strong authentication for client/server applications by using secret-key cryptography. 
-  doc_url: https://web.mit.edu/kerberos/krb5-1.20/doc/index.html
+  doc_url: https://web.mit.edu/kerberos/krb5-1.21/doc/index.html
   dev_url: https://kerberos.org/dist/index.html
```

```suggestion
  dev_url: https://github.com/krb5/krb5
```
Does the GitHub URL make more sense here?